### ISSUE TITLE
IllegalArgumetException wenn Repository Parameter nicht angegeben

### ISSUE THREAD

**Issue by SemikolonDEV** [itplr-kosit/validator#112]
Beim aufruf des Validators ohne Repository fliegt eine IllegalArgumentException. Es kann keine Validierung durchgeführt werden.

Es muss ein Aufruf mit -r "" erfolgen. Obwohl der r Parameter Optional ist.

![image](https://user-images.githubusercontent.com/11752602/230041127-ea0d3378-74a7-4505-af0b-e7f398cf6f7d.png)

**Comment by simon-gotomaxx** [comment]
Ich kann das Problem auch nachvollziehen.

**Comment by landrix** [comment]
Ja, -r ist erforderlich mit dem Hauptpfad zum repository. Also dort wo scenarios.xml liegt

**Comment by phax** [comment]
Das könnte man auch in einem Beispielaufruf darstellen....

**Comment by tobkle** [comment]
hatte das gleiche Problem. Ein Hinweis in der Dokumentation würde helfen.

**Comment by karakoc2107** [comment]
same problems.   Where exactly is the repostyre file?

**Comment by dogonbb** [comment]
It's the location where your scenarios.xml file is stored. So when downloading (https://github.com/itplr-kosit/validator-configuration-xrechnung/blob/master/docs/usage.md), it's saved in the validator-configuration-xrechnung directory.

**Comment by phax** [comment]
Part of Release v1.5.2 from https://github.com/itplr-kosit/validator/releases/tag/v1.5.2
The "-r" parameter is optional again, if only one unnamed scenario is used
